pkg/certifier/osv/osv.go

```diff
@@ -52,10 +55,11 @@ type osvCertifier struct {
 
 // NewOSVCertificationParser initializes the OSVCertifier
 func NewOSVCertificationParser() certifier.Certifier {
+	limiter := rate.NewLimiter(rate.Every(time.Minute), 10000)
```

Comment:
make the number here as a const or configurable?

Reply:
@lumjjb Thanks you for the review, I have made the number a constant for osv, deps dev, and clearly defined.